pipelines.adoc

```diff
@@ -1,266 +1,189 @@
 repo.
 
-Build-deploy-pipeline.yaml
-(https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-deploy-pipeline.yaml)
-is the primary pipeline associated with the collections that will
-validate the collection is active, build the git source, publish the
-container image, and deploy the application. It looks for git-source and
-docker-image resources that will be used by the build-task and
-deploy-task.
-
-Build-task.yaml
-(https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-task.yaml)
-builds a container image from the artifacts in the git-source repository
-using buildah. Once the image is built, it will publish it to the
-docker-image URL using buildah.
-
-Deploy-task.yaml
-(https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-task.yaml)
-will modify the app-deploy.yaml, which describes the deployment options
-for the application, to point to the image that was published and will
-deploy the application using the appsody operator.
-
-For additional tasks and more information about kabanero pipelines,
-visit the kabanero-pipelines
-(https://github.com/kabanero-io/kabanero-pipelines) repo.
+=== The build and deploy pipeline
+
+link:https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-deploy-pipeline.yaml[Build-deploy-pipeline.yaml]
+
+This file is the primary pipeline that is associated with the collections. It validates that the collection is active, builds the Git source, publishes the container image, and deploys the application. It looks for `git-source` and `docker-image` resources that are used by the `build-task` and `deploy-task` files.
+
+=== Tasks
+
+- link:https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-task.yaml[`Build-task.yaml`]
+
+This file builds a container image from the artifacts in the git-source repository by using link:https://github.com/containers/buildah[Buildah]. After the image is built, the `build-task` file publishes it to the docker-image URL by using Buildah.
+
+- link:https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-task.yaml[`Deploy-task.yaml`]
+
+This file modifies the `app-deploy.yaml` file, which describes the deployment options for the application. `Deploy-task` modifies `app-deploy.yaml` to point to the image that was published and deploys the application by using the Appsody operator. By default, the pipelines run and deploy the application in the `+kabanero+` namespace. If you want to deploy the application in a different namespace, update the `app-deploy.yaml` file to point to that namespace.
+
```

Comment:
Maybe we should also add here as a tip or reminder to the user to generate the app-deploy.yaml using appsody deploy --generate-only and to add it to their repo.

Reply:
yes